components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/FederatedAuthenticatorConfig.java

```diff
@@ -101,9 +111,18 @@ public static FederatedAuthenticatorConfig build(OMElement federatedAuthenticato
                     Property[] propertiesArr = propertiesArrList.toArray(new Property[propertiesArrList.size()]);
                     federatedAuthenticatorConfig.setProperties(propertiesArr);
                 }
+            } else if ("DefinedBy".equals(elementName)) {
+                federatedAuthenticatorConfig.setDefinedByType(
+                        IdentityConstants.DefinedByType.valueOf(element.getText()));
             }
         }
 
+        if (federatedAuthenticatorConfig.getDefinedByType() == null) {
+            federatedAuthenticatorConfig.setDefinedByType(IdentityConstants.DefinedByType.SYSTEM);
+            LOG.debug("The defined by type is not set for the : " + federatedAuthenticatorConfig.getName());
```

Comment:
Let's not have this debug log

Reply:
Addressed with commit 2ac85c638a3c93aa5087edea052612887d0b757f